src/storm-dft/parser/BEOrderParser.cpp

```diff
@@ -0,0 +1,58 @@
+#include "BEOrderParser.h"
+
+#include <boost/algorithm/string.hpp>
```

Comment:
Rather than this, could we use c++20 ranges ?

Reply:
I used c++20 ranges now, but it is not as nice as I had hoped.
I could not split on multiple delimiters and I did not find a good replacement for `boost::trim` (but this is not too important as whitespace is removed anyway.)

Happy to receive some suggestions how to do it better, but it should work for now.

Reply:
Using ranges lead to new issues. I opt to keep using boost for now and will open a separate issue.